clang/test/CIR/CodeGen/complex.c

```diff
@@ -0,0 +1,92 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
```

Comment:
Can you please add an extra RUN line that builds this for C++?

Reply:
Updated.